### DISCUSSION TITLE
What do people use for Blazor Wasm Log Provider.

### DISCUSSION THREAD

**Discussion by gallwood** [Megabit/Blazorise#1871] [General]
I was wanting to inject an ILogger in to my pages, but I need to configure a log provider in Program.cs I believe.

I've tried the Blazor.Extensions.Logging v2.0.1 but this throws quite a lot of exceptions when the page loads.

Just wondered what are other people using? I'm expecting to see log message in the Browser Tools -> Console window.

Thanks

Graham

**Comment by stsrki** [comment]
This might help you https://www.thinktecture.com/en/blazor/webassembly-changing-log-level-at-runtime/

**Comment by gallwood** [comment]
Perfect. Thanks
